apps/gitness/src/pages/pull-request/pull-request-changes-page.tsx

```diff
@@ -200,7 +204,7 @@ export default function PullRequestChangesPage() {
         active={''}
         loading={loadingReviewers}
         currentUser={currentUser ?? {}}
-        pullRequestMetadata={pullReqMetadata}
+        pullRequestMetadata={pullReqMetadata ? pullReqMetadata : undefined}
```

Comment:
is this data actually nullable? 
if not, the component should not accept `undefined` values. we should just render the component conditionally.